commit 3e3c3a16b9a8b114cc407da9152e20d5dfc422c6
Author: Andreas Sturmlechner <andreas.sturmlechner@gmail.com>
Date:   Wed Jan 4 14:10:56 2017 +0100

    Make Activities and KF5Html really optional.
    
    Before, trying to use CMAKE_DISABLE_FIND_PACKAGE_KF5{Activities,KHtml}
    errored out with:
    
    "if given arguments: "VERSION_LESS" "5.16.0" Unknown arguments specified"

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0394af4..e11d998 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -223,11 +223,10 @@ find_package(KF5 ${REQUIRED_KF5_VERSION} REQUIRED
         Notifications
         KCMUtils
 )
-find_package(KF5 ${REQUIRED_KF5_VERSION} QUIET
-    OPTIONAL_COMPONENTS
-        Activities
-        KHtml
-)
+
+find_package(KF5Activities)
+find_package(KF5KHtml)
+
 set_package_properties(KF5Activities PROPERTIES
     TYPE OPTIONAL
 )
